main.js

```diff
@@ -37,13 +37,51 @@ class Car {
   }
 }
 
-var $img = document.createElement('img')
-$img.className = 'east'
-$img.src = 'car-top-down.png'
-$img.style = "top: 0px; left: 0px;"
-document.body.appendChild($img)
+class RaceCar extends Car {
+  constructor($img, speed, direction, location, nitros) {
+    super($img, speed, direction, location)
+    this.nitros = nitros
+    this.started = false
+    this.interval = null
+  }
+  nitro() {
+    if (this.nitros > 0) {
+      this.speed *= 2
+      this.nitros--
+    }
+  }
+}
 
-var car = new Car($img, 10, 'east', [0, 0])
+function updateCarObj(type) {
```

Comment:
Improve this function.

* Remove global side effect on car.
* Remove DOM access requirement.
* Return a value